README.md

```diff
@@ -15,6 +15,13 @@ Made with ♥️ by [ChecklyHQ](https://www.checklyhq.com?utm_source=headlessdev
 We welcome new articles on Puppeteer, Playwright and any topic surrounding the practical application of modern headless browser
 frameworks. [Read here how to contribute](CONTRIBUTING.md)
 
+## Authors
+
+- Giovanni Rago ([@rag0g](https://twitter.com/rag0g)) – [Checkly](https://www.checklyhq.com?utm_source=headlessdevgithub)
```

Comment:
@ragog I sneaked another change into this PR. Please review.

Reply:
LGTM